tests/cephadm/test_client.py

```diff
@@ -122,7 +126,12 @@ def setup(host):
                     _node.exec_command(sudo=True, cmd=f"{enable_cmd}{repos}")
 
                 for repos in cdn_ceph_repo[rhel_version][rhcs_version]:
-                    _node.exec_command(sudo=True, cmd=f"{enable_cmd}{repos}")
+                    if rhel_version == "8" and rhcs_version == "6":
```

Comment:
For this specific condition add comment and use repo direclty here.

Reply:
As discussed, Added comment